### PR TITLE
Fix the cli worker

### PR DESCRIPTION
**Pull request by bakura10**
* Fix the CLI worker to also extract the attributes and properly forwarding them like a real worker.
* Add the changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
+# 6.0.0
+
+* [BC] `LocalhostCheckerMiddleware` has been removed. Now, all the security checks are done within the `WorkerMiddleware`. Especially, in addition of
```
Comment (danizord) [inline CHANGELOG.md]: That's BC Break, not BC

Comment (bakura10) [reply CHANGELOG.md]: BC = Breaking compatibility no?

Comment (danizord) [reply CHANGELOG.md]: nope, BC stands for "Backwards Compatibility", so this is a BC break

Comment (bakura10) [reply CHANGELOG.md]: Or Breaking Change? :p Haha so I've misunderstood the acronym for so many time.

Comment (danizord) [reply CHANGELOG.md]: Yep, your entire life was a lie :P

**Pull request merged**